src/app/aws/metrics-baseline.ts

```diff
@@ -43,4 +45,4 @@ export const recordBaselineCloudWatchMetrics = () => {
 			}
 		}
 	});
-};
+}; */
```

Comment:
is this necessary if we're commenting out the import of this file?

Reply:
I've just commented out the whole file to be honest for clarity. There are some calls midway down too that fire off some scheduled jobs which this disables (in addition to the server.ts initialisation). Not actually sure why we have both so need to investigate that.